src/lib/components/End/OnDate.js

```diff
@@ -50,6 +72,7 @@ EndOnDate.propTypes = {
     date: PropTypes.string.isRequired,
     options: PropTypes.shape({
       weekStartsOnSunday: PropTypes.bool,
+      calendarComponent: PropTypes.func,
```

Comment:
It can also be type `element`
check https://reactjs.org/docs/typechecking-with-proptypes.html